presto-pinot/pom.xml

```diff
@@ -30,6 +30,10 @@
                     <groupId>com.google.errorprone</groupId>
                     <artifactId>error_prone_annotations</artifactId>
                 </exclusion>
+                <exclusion>
```

Comment:
Are we confident this dependency is not required? Can we not upgrade the dependency to a newer version without the CVE?

Reply:
I see that the most recent versions of helix-core don't have any CVE - https://mvnrepository.com/artifact/org.apache.helix/helix-core

I also see that newer versions of Pinot are already on one of these versions, we can also explore upgrading Pinot if it's relatively less complex.